http/headers/Accept-Encoding.json

```diff
@@ -4,7 +4,10 @@
       "Accept-Encoding": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Reference/Headers/Accept-Encoding",
-          "spec_url": "https://httpwg.org/specs/rfc9110.html#field.accept-encoding",
+          "spec_url": [
+            "https://httpwg.org/specs/rfc9110.html#field.accept-encoding",
+            "https://www.rfc-editor.org/rfc/rfc8878#name-content-encoding"
```

Comment:
This second spec URL moved here from `zstd`, because it wasn't specific to that value.